tests/functional/pyocf/types/shared.py

```diff
@@ -54,6 +54,19 @@ class OcfCompletion:
     management API.
     """
 
+    class CompletionResult:
+        def __init__(self, completion_args):
+            self.completion_args = completion_args
+            self.results = None
+            self.arg_types = list(list(zip(*self.completion_args))[1])
```

Comment:
`self.arg_types = list(dict(self.completion_args).keys())`
BTW why isn't `completion_args` a dictionary? That would simplify `__getitem__` as well.

Reply:
Yep, it can be a dictionary. Initially I thought I needed it that way to preserve order of arguments, but in fact I can just store the order in dictionary too :)